enterprise/models.py

```diff
@@ -9,12 +9,15 @@
 from uuid import uuid4
 
 import six
+from requests.compat import urljoin
```

Comment:
I'd prefer to import `urljoin` from `six.moves.urllib.parse`; `requests.compat` is an internal module for use specifically inside `requests`, whereas `six` is all about providing cross-Python-version compatibility.